test/e2e/config/docker.yaml

```diff
@@ -35,13 +35,13 @@ providers:
     replacements:
     - old: --metrics-addr=127.0.0.1:8080
       new: --metrics-addr=:8080
```

Comment:
@fabriziopandini  Not sure if we want to change v0.3.23 to v0.4.2 or add v0.4.2.

But doesn't have to be done in this PR anyway